frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -1,19 +1,33 @@
 <template>
   <div class="transaction-link gradido-custom-background">
     <b-row class="mb-2 pt-2 pb-2">
-      <b-col cols="2">
+      <b-col lg="2">
         <type-icon color="text-danger" icon="link45deg" class="pt-4 pl-2" />
       </b-col>
-      <b-col cols="9">
-        <amount-and-name-row :amount="amount" :text="$t('form.amount')" />
-        <memo-row :memo="memo" />
-        <date-row :date="validUntil" :diffNow="true" />
-        <decay-row :decay="decay" />
+      <b-col lg="9" md="9">
+        <b-row>
+          <b-col lg="11" md="10">
+            <amount-and-name-row :amount="amount" :text="$t('form.amount')" />
+            <memo-row :memo="memo" />
+            <date-row :date="validUntil" :diffNow="true" />
+            <decay-row :decay="decay" />
+          </b-col>
+          <b-col lg="1" md="2" class="text-center text-lg-left qr-button">
+            <b-button
+              @click="$bvModal.show('modalPopover-' + id)"
+              class="p-2 test-qr-code"
+              size="sm"
+              variant="outline-success"
+            >
+              <b-img src="img/svg/qr-code.svg" width="60"></b-img>
+            </b-button>
+          </b-col>
+        </b-row>
       </b-col>
 
-      <b-col cols="1" class="text-right">
+      <b-col lg="1" md="1" class="text-center text-lg-right">
         <b-button
-          class="p-2"
+          class="p-2 test-copy-link"
```

Comment:
are you sure you want this name on the element? Why not find the button as before?

Reply:
weil mich es stört das wenn man die button position verändert der gesamte test einem um die ohren fliegt. weil at() nicht mehr passt